src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext, useState } from 'react';
+import classNames from 'classnames';
+
+import { TodoContext } from '../TodoContext';
+
+export const Notification: React.FC = () => {
+  const { errorMessage } = useContext(TodoContext);
+  const [isHidden, setIsHidden] = useState(false);
+
+  setTimeout(() => {
+    setIsHidden(true);
+  }, 3000);
+
+  return (
+    <>
+      {errorMessage && (
```

Comment:
you don't need conditional rendering inside this component since you've already checked it in parent component